.github/CONTRIBUTING.md

```diff
@@ -67,95 +60,94 @@ If you haven't used git before, the GitHub website and GitHub desktop client all
 Any file can be edited in the GitHub interface, and new files can be created. 
 GitHub will create these as a new pull request.
 
-### Using Git at the Command Line
+### Using Git at the Command Line for GitHub
```

Comment:
* a lot of this is specific to the issues-and-projects repository even though the PR template is for the entire system
* At this point there are so many tutorials on using git ... would it be sufficient to link to documentation maintained by github, e.g. https://guides.github.com/activities/forking/ or this course https://lab.github.com/githubtraining/introduction-to-github? or, should this perhaps go in the OSF page about Github?

Reply:
This is out of scope for the issue being merged: https://app.zenhub.com/workspaces/ua-ag-data-science-5a57a3198339f11ba1c85775/issues/agpipeline/issues-and-projects/186

Updated PR to reflect which issue it's tied to.

I will create a ticket for this comment: https://app.zenhub.com/workspaces/ua-ag-data-science-5a57a3198339f11ba1c85775/issues/agpipeline/issues-and-projects/218

Reply:
Also added an issue for adding a section to the PR template for linking to the issue a PR is for: https://app.zenhub.com/workspaces/ua-ag-data-science-5a57a3198339f11ba1c85775/issues/agpipeline/issues-and-projects/219